### PR TITLE
160: Migrate to ResourceLoader

### PR DESCRIPTION
**Pull request by abdullahQureshee**
# Overview
Resolves https://github.com/openedx/xblock-qualtrics-survey/issues/160

# Test Instructions
- Checkout the branch
- make test

# TODO
- [x] Lint all files
- [x] Pass all tests
- [x] Bump the version number in `setup.py`
- [x] Attach screenshots?

# Screenshots

## cms
Screenshot 2024-09-16 at 3 38 52 PM

## lms
Screenshot 2024-09-16 at 3 39 48 PM

### CODE REVIEW

**Comment by farhan** [comment]
Update the major version to release this xblock

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = csslint,eslint,py38-django{32,42},quality
+envlist = csslint,eslint,py311-django{42},py312-django{42},quality
```
Comment (farhan) [inline tox.ini:2]: Update to
```yml
csslint,eslint,py{311,312}-django{42},quality
```

File: setup.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from setuptools import find_packages, setup
 
 
-version = '1.4.1'
+version = '1.5.1'
```
Comment (farhan) [inline setup.py:11]: Update major version as we are dropping support of Python 3.8

**Review by farhan — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (farhan) [reply setup.py:11]: Please update it to 2.0.0

**Review by farhan — APPROVED** [review]

**Review by feanil — APPROVED** [review]

**Pull request merged**